apps/dialtone-documentation/docs/components/empty-states.md

```diff
@@ -0,0 +1,378 @@
+- Secondary*
+
+5.Background
+
+*optionals
+
+The empty state uses all the space of the container it's replacing, always keeping the content in the middle.
+
+The space taken by the empty state can vary, the padding will be minimum 24px and can be flexible and adapt the container size.
+
+## Uses & Best practices
+
+We identify 4 types of empty states:
+
+<b> Zero state: </b> nothing has been added yet, we recommend to use this space to teach the user how to fill this container, or give a useful tip like a shortcut for something relatable or tell them what needs to happen if no action can be taken.
+
+<img class="d-w90p" alt="empty state example" src="/assets/images/components/empty-state02.png">
+
+<b> No results:</b> the user performed an action and there are no results, in this scenario the best usage of the empty state will be to recommend how to recover. If the user is filtering a table, recommend another filter that has results or, like the zero state case, how to create a new entry with the properties the user is filtering.
+
+<img class="d-w90p" alt="empty state example" src="/assets/images/components/empty-state03.png">
+
+<b> New feature:</b> A new section or screen the user doesn’t know and we want to introduce to them, if this action would include a change in their plan we should inform it in the component, also try to indulge/seduce the user to use this feature or to look interesting to them.
+
+<img class="d-w90p" alt="empty state example" src="/assets/images/components/empty-state04.png">
+
+<b> Not enabled:</b> the section isn’t available to the user at the moment.
+
+<img class="d-w90p" alt="empty state example" src="/assets/images/components/empty-state05.png">
+
+<dialtone-usage>
+<template #do>
+
+- Give the user all the information necessary to know how to proceed, but not as much for them to feel overwhelmed.
+- If you are gonna use an illustration or an icon, try to have it in a 2nd hierarchical level, to avoid distracting the user from the real task, we recommend to use desaturated color’s  illustrations and keep the text as a main thing the user should read.
+- The background of the component should match the background of the environment
+- The component can contain borders, depending on the context, see examples below.
+
+</template>
+
+<template #dont>
+
+- Use the empty state as a dead end, always keep the user options on what to do.
+- Use illustrations as a main part of the component
+- Use elements too colorful to take away the attention from the main task
+</template>
+
+</dialtone-usage>
+
+### Writing and tone
+
+We think this component is one of the most important regarding the [writing guidelines](/guides/writing-guidelines/index.md) (along with the error messages) so we take some time to write about how to communicate with our empty states.
+First of all, let’s talk about the different atoms that can have a writing variable
+
+- Titles: Needs to be scannable and informative, keep it short and concise.
```

Comment:
Will make it a bit more readable if you bold these prefixes ex "Titles:" "Body:"